package.json

```diff
@@ -18,11 +18,20 @@
   },
   "homepage": "https://github.com/GSG-K3/ca-application-website#readme",
   "devDependencies": {
+    "bcryptjs": "^2.4.3",
```

Comment:
Is "bcryptjs"  the same package as  "bcrypt"? since I added the last one to the previous setup